docs/02-iaas/default-security-group-rules.md

```diff
@@ -0,0 +1,89 @@
+
+## How to comply to the standard for default Security Group Rules
+
+The standard for default Security Group Rules does not allow any ingress traffic rules, but keeps egress rules as optional.
+
+To test what default Security Group rules are currently in place this command can be used:
+
+```bash
+openstack default security group rule list
+```
+
+If there are any ingress rules in this set of rules, administrators MUST delete them with the following command:
+
+```bash
+openstack default security group rule delete $DEFAULT_SG_RULE_ID
+```
+
+Egress rules can also be deleted this way, if security guidelines specify it.
+
+### Example of a conform set of default Security Group Rules
+
+This is an exapmle how a table with default Security Group Rules may look like:
+
+```bash
+$ openstack default security group rule list
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+| ID                       | IP Protocol | Ethertype | IP Range  | Port Range | Direction | Remote Security Group | Remote Address Group | Used in default Security Group | Used in custom Security Group |
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+| 47b929fd-9b39-4f22-abc5- | None        | IPv6      | ::/0      |            | egress    | None                  | None                 | True                           | True                          |
+| 7d4f64d10909             |             |           |           |            |           |                       |                      |                                |                               |
+| 92a79600-5358-4fef-a390- | None        | IPv4      | 0.0.0.0/0 |            | egress    | None                  | None                 | True                           | True                          |
+| 822665f46070             |             |           |           |            |           |                       |                      |                                |                               |
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+```
+
+## Other possible configuration options with default Security Group Rules
+
+:::tip
+
+Creating default Security Group Rules works in a similiar way as creating rules for one Security Group.
+The parameters that can be used to describe rules are mostly the same.
+
+:::
+
+The default security Group Rules can differ between default Security Groups and all other Security Groups - called `custom Security Group`.
```

Comment:
```suggestion
Default Security Group Rules may be adjusted individually for one of two different types of Security Groups:

1. The default Security Groups that are automatically created for projects initially.
2. Custom Security Groups created by users and initialized with a default rule set.

The default rule set is configured separately for each of those two types.
```

I found the original sentence a bit hard to grasp due to the meaning of "default" and "custom" being separate between rules and groups making this multi-dimensional but hard to convey in a single sentence. That's the reason I rephrased this to make a more clear distinction.